<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IM View</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Images.qrc">
    <normaloff>:/icons/data/img/icons/IM_48x48.png</normaloff>:/icons/data/img/icons/IM_48x48.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Исходные данные</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="datas" name="widget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <attribute name="title">
        <string>Идентификация параметров</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="identf" name="widget_2" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Энергетические показатели</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="electromagn" name="widget_3" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Тепловая модель</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="teplovent" name="widget_5" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Вентиляционная модель</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="vent_model" name="widget_6" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_6">
       <attribute name="title">
        <string>Выходные данные</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="Itogs" name="widget_4" native="true">
          <layout class="QGridLayout" name="gridLayout_2"/>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1116</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>&amp;Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
    <addaction name="action_6"/>
    <addaction name="separator"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuabout">
    <property name="title">
     <string>&amp;Справка</string>
    </property>
    <addaction name="actionhelp"/>
    <addaction name="actionabout"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>&amp;Инструменты</string>
    </property>
    <addaction name="action_15"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>&amp;Правка</string>
    </property>
    <addaction name="action_7"/>
    <addaction name="action_8"/>
    <addaction name="separator"/>
    <addaction name="actioncut"/>
    <addaction name="action_10"/>
    <addaction name="action_11"/>
    <addaction name="action_4"/>
    <addaction name="separator"/>
    <addaction name="action_13"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>&amp;Идентификация параметров</string>
    </property>
    <addaction name="action_5"/>
    <addaction name="action_9"/>
    <addaction name="separator"/>
    <addaction name="action_14"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Исходные данные</string>
    </property>
    <addaction name="action_12"/>
    <addaction name="action_16"/>
    <addaction name="action_17"/>
    <addaction name="action_18"/>
    <addaction name="action_19"/>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>&amp;Энергетические показатели</string>
    </property>
    <addaction name="action_20"/>
    <addaction name="action_21"/>
    <addaction name="action_22"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menu_2"/>
   <addaction name="menu_4"/>
   <addaction name="menu_3"/>
   <addaction name="menu_5"/>
   <addaction name="menu"/>
   <addaction name="menuabout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action"/>
   <addaction name="action_2"/>
   <addaction name="action_6"/>
   <addaction name="separator"/>
   <addaction name="action_3"/>
   <addaction name="separator"/>
   <addaction name="actioncut"/>
   <addaction name="action_10"/>
   <addaction name="action_11"/>
   <addaction name="separator"/>
   <addaction name="action_5"/>
   <addaction name="action_9"/>
   <addaction name="separator"/>
   <addaction name="action_20"/>
   <addaction name="action_21"/>
   <addaction name="action_22"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/application-exit.svg</normaloff>:/system_icons/data/img/system_icons/application-exit.svg</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/system-help.svg</normaloff>:/system_icons/data/img/system_icons/system-help.svg</iconset>
   </property>
   <property name="text">
    <string>Справка</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionabout">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/help-about.svg</normaloff>:/system_icons/data/img/system_icons/help-about.svg</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/document-open.svg</normaloff>:/system_icons/data/img/system_icons/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/document-save.svg</normaloff>:/system_icons/data/img/system_icons/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/document-print.svg</normaloff>:/system_icons/data/img/system_icons/document-print.svg</iconset>
   </property>
   <property name="text">
    <string>Печать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/document-save-as.svg</normaloff>:/system_icons/data/img/system_icons/document-save-as.svg</iconset>
   </property>
   <property name="text">
    <string>Сохранить рисунок</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/edit-undo.svg</normaloff>:/system_icons/data/img/system_icons/edit-undo.svg</iconset>
   </property>
   <property name="text">
    <string>Отменить действие</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_8">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/edit-redo.svg</normaloff>:/system_icons/data/img/system_icons/edit-redo.svg</iconset>
   </property>
   <property name="text">
    <string>Повторить действие</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actioncut">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/edit-cut.svg</normaloff>:/system_icons/data/img/system_icons/edit-cut.svg</iconset>
   </property>
   <property name="text">
    <string>Вырезать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_10">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/edit-copy.svg</normaloff>:/system_icons/data/img/system_icons/edit-copy.svg</iconset>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_11">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/edit-paste.svg</normaloff>:/system_icons/data/img/system_icons/edit-paste.svg</iconset>
   </property>
   <property name="text">
    <string>Вставить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_13">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/edit-find.svg</normaloff>:/system_icons/data/img/system_icons/edit-find.svg</iconset>
   </property>
   <property name="text">
    <string>Поиск/замена</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/edit-delete.svg</normaloff>:/system_icons/data/img/system_icons/edit-delete.svg</iconset>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/media-playback-start_2.svg</normaloff>:/system_icons/data/img/system_icons/media-playback-start_2.svg</iconset>
   </property>
   <property name="text">
    <string>Пуск</string>
   </property>
  </action>
  <action name="action_9">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/media-playback-stop_2.svg</normaloff>:/system_icons/data/img/system_icons/media-playback-stop_2.svg</iconset>
   </property>
   <property name="text">
    <string>Стоп</string>
   </property>
  </action>
  <action name="action_14">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/configure.svg</normaloff>:/system_icons/data/img/system_icons/configure.svg</iconset>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="action_15">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/configure.svg</normaloff>:/system_icons/data/img/system_icons/configure.svg</iconset>
   </property>
   <property name="text">
    <string>Настройки</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_12">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/document-open.svg</normaloff>:/system_icons/data/img/system_icons/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>Загрузить данные</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_16">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/document-save.svg</normaloff>:/system_icons/data/img/system_icons/document-save.svg</iconset>
   </property>
   <property name="text">
    <string>Сохранить данные</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_17">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/edit-delete.svg</normaloff>:/system_icons/data/img/system_icons/edit-delete.svg</iconset>
   </property>
   <property name="text">
    <string>Удалить данные</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_18">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/edit-clear.svg</normaloff>:/system_icons/data/img/system_icons/edit-clear.svg</iconset>
   </property>
   <property name="text">
    <string>Очистить данные</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_19">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/document-new.svg</normaloff>:/system_icons/data/img/system_icons/document-new.svg</iconset>
   </property>
   <property name="text">
    <string>Принять данные</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_20">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/media-playback-start.svg</normaloff>:/system_icons/data/img/system_icons/media-playback-start.svg</iconset>
   </property>
   <property name="text">
    <string>Запуск</string>
   </property>
  </action>
  <action name="action_21">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/media-playback-stop.svg</normaloff>:/system_icons/data/img/system_icons/media-playback-stop.svg</iconset>
   </property>
   <property name="text">
    <string>Останов</string>
   </property>
  </action>
  <action name="action_22">
   <property name="icon">
    <iconset resource="Images.qrc">
     <normaloff>:/system_icons/data/img/system_icons/view-refresh.svg</normaloff>:/system_icons/data/img/system_icons/view-refresh.svg</iconset>
   </property>
   <property name="text">
    <string>Калибровка осциллографа</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>datas</class>
   <extends>QWidget</extends>
   <header>datas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>identf</class>
   <extends>QWidget</extends>
   <header>identf.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>electromagn</class>
   <extends>QWidget</extends>
   <header>electromagn.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Itogs</class>
   <extends>QWidget</extends>
   <header>itogs.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>teplovent</class>
   <extends>QWidget</extends>
   <header>teplovent.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>vent_model</class>
   <extends>QWidget</extends>
   <header>vent_model.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
